app/assets/stylesheets/styles/admin.scss

```diff
@@ -0,0 +1,33 @@
+
+.container-large {
+  padding-left: 20em;
```

Comment:
Do you mean hyphens(-) or underscores(_)? I have looked at the other scss in the project and they are using underscores